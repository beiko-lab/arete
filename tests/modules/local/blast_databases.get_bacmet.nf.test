nextflow_process {

    name "Test Process GET_BACMET"
    script "modules/local/blast_databases.nf"
    process "GET_BACMET"

    test("Can download BacMet") {

        then {
            assert process.success
            assert process.trace.tasks().size() == 1
            assert snapshot(process.out).match()
            with(process.out) {
              // check if emitted output has been created
              assert bacmet.size() == 1
            }
        }

    }

}
