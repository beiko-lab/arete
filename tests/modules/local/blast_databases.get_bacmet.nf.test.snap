{
    "Can download BacMet": {
        "content": [
            {
                "bacmet": [
                    "BacMet2_predicted_database.fasta.gz:md5,d736989efea2594aad1daf548ece8801"
                ]
            }
        ],
        "timestamp": "2023-02-03T14:43:02+0000"
    }
}